util/archutil/Dockerfile

```diff
@@ -8,7 +8,11 @@ RUN apt-get update && apt-get --no-install-recommends install -y \
   binutils-s390x-linux-gnu \
   binutils-powerpc64le-linux-gnu \
   binutils-mips64el-linux-gnuabi64 \
-  binutils-mips64-linux-gnuabi64
+  binutils-mips64-linux-gnuabi64 \
+  wget xz-utils 
+RUN wget http://ftp.loongnix.cn/toolchain/gcc/release/loongarch/gcc8/loongson-gnu-toolchain-8.3-x86_64-loongarch64-linux-gnu-rc1.2.tar.xz \
```

Comment:
This should be installed from the source until Debian provides `binutils-loongarch64-linux-gnu`.
It should be also opt-in via build-arg.

Reply:
bit of a side-side, but if we add this, could this be a good opportunity to use `ADD --checksum` ?

Reply:
@AkihiroSuda Loongarch is about to be supported by Debian. In July, the official community has announced that Loongarch will become the next goal after RISCV64. This will be temporary

Reply:
@yzewei Looks like some packages do exists in ports (I don't know if stable though). Maybe you can start from adding support in https://github.com/tonistiigi/xx . We also build binutils there https://github.com/tonistiigi/xx/blob/master/src/ld/Dockerfile#L234 for arch that are supported in `xx`. Eg. this is how ports is used for riscv https://github.com/tonistiigi/xx/blob/master/src/xx-apt#L140

Reply:
> bit of a side-side, but if we add this, could this be a good opportunity to use `ADD --checksum` ?
Do you mean to add SHA to ensure the reliability of the installation package?

Reply:
@tonistiigi Thank you very much for your reply!
First , Binutils originated from Loongarch official, its security is guaranteed
Secondly,  According to your reply, we can submit the PR supported by LOONGARCH under the [[XX project]](https://github.com/tonistiigi/xx)(https://github.com/tonistiigi/xx )
Thanks again!

Reply:
Maybe we can use trixie and call it a day
https://packages.debian.org/trixie/binutils-loongarch64-linux-gnu

Reply:
@AkihiroSuda  Can I switch the mirror with **trixie**? Thanks